.github/workflows/continous-integration.yml

```diff
@@ -71,6 +72,7 @@ jobs:
     - uses: actions/setup-python@v2
       with:
         python-version: 3.8
+        cache: 'pip'
```

Comment:
According to https://github.com/actions/setup-python, we can set the "cache" flag, which uses https://github.com/actions/toolkit/tree/main/packages/cache under the hood!